plutus-core/plutus-ir/test/transform/retainedSize/datatypeDead.golden

```diff
@@ -0,0 +1,13 @@
+(let
+  (nonrec)
+  (datatypebind
+    (datatype
+      $1$
+      (tyvardecl $4$ Maybe (fun (type) (type)))
+      (tyvardecl $17$ a (type))
+      match_Maybe
+      (vardecl $4$ Nothing [ Maybe a ]) (vardecl $6$ Just (fun a [ Maybe a ]))
+    )
+  )
+  (abs a (type) (lam x a x))
+)
```

Comment:
But all of these are also not referenced in the body of the let and yet they're shown to retain whatever each of them retains directly. Can't explain that.

Reply:
Oh, wait, I think I see the problem.

Reply:
Fixed, I forgot to rename the program before performing retained size analysis, so `a`-the-parameter and `a`-the-abs-bound-variable got confused.

Reply:
Hm, it still shows the wrong thing...

Reply:
Ah, no, that's just the old diff, [the file is fine](https://github.com/input-output-hk/plutus/blob/db2b3f5e6720459721d183f322684cec25e87dc8/plutus-core/plutus-ir/test/transform/retainedSize/datatypeDead.golden).